### PR TITLE
fix: style of the projectName so that it is visible completely

### LINKED ISSUE #3387: bug: Project Name and User Name is not visible completely
**Describe the bug**
Bug1           |  Bug2 
:-------------------------:|:-------------------------:
  |  

- Bug1 - Below part of the project name got hidden. For example - `y` is not fully visible as you can see in ss
- Bug2 - Even after zooming out bug1 exist. Also, the Project name alignment breaks and got aligned to the left side a bit instead of being center-aligned.

**To Reproduce**
1. Go to 'Simulator'
2. Click 'Untitled'
3. Change 'Untitled' to 'Binary to Gray' (for ex.)
4. See the error

**Expected behavior**
Bug1      |  Bug2 
:-------------------------:|:-------------------------:
  |  

**Screenshots**

**Device Information [optional]:**
 - OS: Windows, Linux
 - Browser - Firefox

**Additional context**

**Are you working on this issue? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by Waishnav**
Fixes #3387

#### Describe the changes you have made in this PR -
Made Username and project name fully visible. And Solved these two small bugs through refactoring.

### Screenshots of the changes (If any) -
Bug1
Before      |  After 
:-------------------------:|:-------------------------:
  |  

Bug2
Before            |  After
:-------------------------:|:-------------------------:
  |  

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: app/views/simulator/edit.html.erb [hunk 1]
```diff
@@ -16,7 +16,7 @@
   <button class="navbar-toggler collapsed" type="button" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-controls="collapsedNavbar" aria-expanded="false" aria-label="Toggle navigation">
     <span class="navbar-toggler-icon"></span>
   </button>
-  <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+  <div class="collapse navbar-collapse" style="justify-content: space-between;" id="bs-example-navbar-collapse-1">
```
Comment (tachyons) [inline app/views/simulator/edit.html.erb:19]: No inline style please

Comment (Waishnav) [reply app/views/simulator/edit.html.erb]: Alright. I thought it was allowed to use. 
It's resolved now using the bootstrap class. Can you please review it again?

**Review by gr455 — APPROVED** [review]
LGTM

**Comment by pulkit-30** [comment]
Not able to reproduce on MacOs

Screenshot 2022-12-13 at 7 51 49 AM

**Comment by Waishnav** [comment]
I did not test on macOS, am the issue was only reproducible on Windows and Linux in my experience.

**Pull request merged**